snuba/web/db_query.py

```diff
@@ -55,6 +57,152 @@ def update_query_metadata_and_stats(
     return stats
 
 
+def execute_query(
+    request: Request,
+    query: ClickhouseQuery,
+    timer: Timer,
+    stats: MutableMapping[str, Any],
+    query_settings: MutableMapping[str, Any],
+    query_id: Optional[str],
+) -> Result:
+    """
+    Execute a query and return a result.
+    """
+    # Experiment, if we are going to grab more than X columns worth of data,
```

Comment:
This is probably no longer an experiment, but I just carried over the existing comment almost word-for-word.